website/docs/r/hpc_cache.html.markdown

```diff
@@ -83,7 +83,15 @@ The following arguments are supported:
 * `directory_ldap` - (Optional) A `directory_ldap` block as defined below.
 
 ~> **Note:** Only one of `directory_active_directory`, `directory_flat_file` and `directory_ldap` can be set.
- 
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `key_vault_key_id` - (Optional) Specifies the URL to a Key Vault Key.
```

Comment:
is this the Resource Manager or Data Plane Key ID?

Reply:
this is Resource Managed ID, which is `azurerm_key_vault_key.id`, have updated the description to be clear